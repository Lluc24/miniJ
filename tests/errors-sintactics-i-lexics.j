5 =< 5
+ 1 2 3
:+ 1 2 3
=: # ] @ 1 2 3 + ]
x := 1
1 2 3 $ 1 2 3
z + 4
