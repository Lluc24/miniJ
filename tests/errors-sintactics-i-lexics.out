8 errors lèxics/sintàctics.
(root (gen (expr (expr 5) (operBin =) (expr operUn (expr (operUn <) (expr 5))))) \n (gen (expr operUn (expr (operUn +) (expr 1 2 3)))) \n : (gen (expr operUn (expr (operUn +) (expr 1 2 3)))) \n =: (gen (expr (operUn #) (expr (operUn ]) (expr @ 1 2 3)))) <missing '\n'> (gen (expr operUn (expr (operUn +) (expr (operUn ]) (expr \n x))))) : = 1 \n 1 2 3 $ 1 2 3 \n z + 4 \n)
